### DISCUSSION TITLE
Whiteboard sharing

### DISCUSSION THREAD

**Discussion by bluemeanie76** [Azure/communication-ui-library#3375] [General]
Is there a way of enabling Microsoft Whiteboard sharing using ACS?

**Comment by bluemeanie76** [comment]
Can anyone provide a response to this?

**Reply by alkwa-msft** [reply]
@bluemeanie76 sorry we don't get monitoring here like we do for Github issues. This is currently on our backlog but we still have a few technical investigations to try out before we can share more about Whiteboard support in Azure Communication Services.
